packages/@webex/internal-plugin-metrics/test/unit/spec/business/business-metrics.ts

```diff
@@ -154,11 +154,12 @@ describe('internal-plugin-metrics', () => {
           businessMetrics.clientMetricsBatcher.request = request;
           
           assert.equal(requestCalls.length, 0)
-          businessMetrics.submitBusinessEvent({ name: "foobar", payload: {bar:"gee"}, table: 'business_metrics' })
+          businessMetrics.submitBusinessEvent({ name: "foobar", payload: {bar: "gee"}, table: 'business_metrics', metadata: {asdf: 'hjkl'} })
           assert.equal(requestCalls.length, 1)
           assert.deepEqual(requestCalls[0], {
             eventPayload: {
               key: 'foobar',
+              asdf: 'hjkl',
```

Comment:
So, if the given metadata field already exists in the request, then this input will override it?

Reply:
Yep, that's right